exporter/googlecloudexporter/config.go

```diff
@@ -0,0 +1,175 @@
+// Copyright  observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package googlecloudexporter
+
+import (
+	"os"
+
+	"github.com/GoogleCloudPlatform/opentelemetry-operations-collector/processor/normalizesumsprocessor"
+	"github.com/mitchellh/mapstructure"
+	"github.com/observiq/observiq-otel-collector/processor/resourceattributetransposerprocessor"
+	gcp "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudexporter"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor"
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourceprocessor"
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/processor/batchprocessor"
+	"go.uber.org/multierr"
+)
+
+const (
+	defaultMetricPrefix = "workloads.googleapis.com"
+	defaultUserAgent    = "observIQ-otel-collector"
+	defaultLocation     = "us-east1-b"
```

Comment:
IMO, we should use `global` as the location, it's just more generic and is what's currently in our examples that map to generic_node.

Reply:
Confirmed, global is working well. We were not aware of global previously.

Reply:
Can do. I was not aware that global was an option.